crates/re_viewer/src/ui/override_ui.rs

```diff
@@ -0,0 +1,266 @@
+                            StoreKind::Blueprint => {
+                                let store = ctx.store_context.blueprint.store();
+                                let query = ctx.blueprint_query;
+                                get_component_with_instances(
+                                    store,
+                                    query,
+                                    entity_path,
+                                    *component_name,
+                                )
+                            }
+                            StoreKind::Recording => get_component_with_instances(
+                                store,
+                                &query,
+                                entity_path,
+                                *component_name,
+                            ),
+                        };
+
+                        if let Some((_, _, component_data)) = component_data {
+                            ctx.component_ui_registry.edit(
+                                ctx,
+                                ui,
+                                UiVerbosity::Small,
+                                &query,
+                                store,
+                                entity_path,
+                                &overrides.override_path,
+                                &component_data,
+                                instance_key,
+                            );
+                        } else {
+                            // TODO(jleibs): This shouldn't happen. Warn instead?
+                            ui.weak("(empty)");
+                        }
+                    });
+                }
+            });
+        });
+}
+
+pub fn add_new_override(
+    ctx: &ViewerContext<'_>,
+    query: &LatestAtQuery,
+    store: &DataStore,
+    ui: &mut egui::Ui,
+    space_view: &SpaceViewBlueprint,
+    data_result: &DataResult,
+    active_components: &BTreeSet<ComponentName>,
+) {
+    ui.menu_button("Add new override", |ui| {
+        ui.style_mut().wrap = Some(false);
+
+        let view_systems = ctx
+            .space_view_class_registry
+            .new_visualizer_collection(*space_view.class_identifier());
+
+        // We have to have at least 1 visualizer system or we can't create an override
+        let Some(system_for_initial_value) = view_systems.systems.values().next() else {
+            return;
+        };
+
+        let mut components_per_visualizer = data_result
+            .visualizers
+            .iter()
+            .filter_map(|vis| view_systems.get_by_identifier(*vis).ok())
+            .map(|vis| vis.visualizer_query_info().queried);
+
+        // Accmulate all the components if there are multiple visualizers
+        let mut components = components_per_visualizer.next().unwrap_or_default();
+        for mut rest in components_per_visualizer {
+            components.append(&mut rest);
+        }
+
+        // Empty space views of every available types
```

Comment:
space views? I presume you mean components

Reply:
No idea what this comment was referring to -- maybe a bad copy-paste from somewhere :man_shrugging: